distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -577,6 +585,25 @@ public void testCswXmlIngest() {
     }
 
     @Test
+    public void testCswXmlWithHeaderIngest() {
+        Response response = ingestXmlWithHeaderMetacard();
+        ValidatableResponse validatableResponse = response.then();
+
+        validatableResponse.body(hasXPath("//TransactionResponse/TransactionSummary/totalInserted",
```

Comment:
Nit: could we just do `response.body(hasXPath...` to remove an unnecessary declaration?